packages/bootstrap/scss/appbar/_index.scss

```diff
@@ -0,0 +1,13 @@
+@import "../_variables.scss";
+@import "../mixins/_index.scss";
+
+
+// Dependencies
+@import "../common/_index.scss";
+@import "../button/_index.scss";
```

Comment:
Are the button and icon an explicit part of the appbar?

Reply:
no, they aren't. I'll remove them.